recipes/tpp/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+BUILD_DIR="${SRC_DIR}/build"
+
+export CXXFLAGS="${CXXFLAGS} -fPIC -w"
+export CFLAGS="-I$PREFIX/include"
+export CPATH=${PREFIX}/include
+export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+
+# TODO: mz5 support is nearly working but needs a few fixes in linking certain
+# targets
+#export HDF5_SUPPORT = 1
+#export MZ5_SUPPORT  = 1
+
+echo "INSTALL_DIR  = ${PREFIX}"      >  site.mk
+echo "TPP_BASEURL  = /tpp"           >> site.mk
+echo "TPP_DATADIR  = ${PREFIX}/data" >> site.mk
+
+# install the dependencies
+
+cpanm FindBin::libs
```

Comment:
This should be a conda dependency as well.

Reply:
Okay, I will have to package it separately

Reply:
Should this go in bioconda or another conda channel? It's not specific to biology but I see other general-purpose perl packages in bioconda.

Reply:
BioConda is fine for the moment I think. We did not mass-migrate perl stuff to conda-forge.